src/web/mjs/connectors/SakuraNovel.mjs

```diff
@@ -0,0 +1,81 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+//template/theme LightNovel, basically WordPressMangastreamNovel
+export default class SakuraNovel extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'sakuranovel';
+        super.label = 'SakuraNovel';
+        this.tags = ['novel', 'indonesian'];
+        this.url = 'https://sakuranovel.id';
+
+        this.querMangaTitleFromURI = 'div.series-title h2';
+        this.queryMangas = 'div.novellist-blc ul li a.series';
+        this.queryChapters = 'ul.series-chapterlists li a';
+        this.queryChaptersTitle = 'span:not(.date)';
+        this.path = '/daftar-novel/';
+
+        this.novelContentQuery = 'div.asdasd';
+        this.novelObstaclesQuery = 'div[class]';
+        this.novelFormat = 'image/png';
+        this.novelWidth = '56em'; // parseInt(1200 / window.devicePixelRatio) + 'px';
+        this.novelPadding = '1.5em';
+    }
+
+    async _getChapters(manga) {
+        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryChapters);
+        return data.map(element => {
+            this.adLinkDecrypt(element);
+            const title = this.queryChaptersTitle ? [...element.querySelectorAll(this.queryChaptersTitle)].map(ele => ele.textContent).join(' ') : element.text;
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: title.replace(manga.title + ' – ', '').replace(/Bahasa Indonesia$/i, '').trim()
```

Comment:
I ask myself why do they need to put the language everywhere.. More garbage, more and mooore garbage..
Anyway,  chapters still got manga title for some reasons according to the screenshot i took.

Did you test? Its enough to use super._getchapters() and then remove bahasa, EXACTLY like you did on monzee for mangas. No need for that complicated stuff.